src/cameras/dtos/create_writer_options.py

```diff
@@ -5,16 +5,16 @@
 
 
 class CreateWriterOptions(BaseModel):
-    filename_and_ext: str
-    parent_folder: str
+    writer_dir: Path
 
 
 def get_canonical_time_str():
-    return time.strftime("%Y%m%d_%H%M%S")
+    return time.strftime("%m_%d_%Y_%H%M%S")
```

Comment:
This is a nice convenience function for getting a time string with Month_Day_Year_Hour_Minute_Second. We use this in the codebase to simplify video paths